benchmark/utils/CxxTests/cxx-types.h

```diff
@@ -0,0 +1,3 @@
+struct CxxIntBox {
```

Comment:
I'd suggest to imitate what we do in the `test/Interop` directory, and create a separate supporting header for each benchmark; otherwise it would be very difficult to untangle supporting code that gets reused between benchmarks that was not actually meant to be reused.

Reply:
Hmm, good idea. I'll do that.